components/breadcrumb/stories/template.js

```diff
@@ -48,9 +47,13 @@ export const Template = ({
 								idx !== arr.length - 1,
 								() =>
 									html`<div
-										class="${rootClass}-itemLink"
+										class=${classMap({
+											[`${rootClass}-itemLink`]: true,
+											"is-disabled": isDisabled,
+										})}
+										?disabled=${isDisabled}
```

Comment:
```suggestion
										aria-disabled=${ifDefined(isDisabled ? "true" : undefined)}
```
The `disabled` attribute does not apply to a div. On the docs site, this has `aria-disabled`, so using that instead would match the existing markup. _Though I wonder why a [disabled anchor tag](https://www.scottohara.me/blog/2021/05/28/disabled-links.html) is not being used instead of a div in the original markup._

Reply:
Yes I wondered about the `role="link"` divs too.

For now, I [changed `disabled` in the template to `aria-disabled`](https://github.com/adobe/spectrum-css/pull/2886/commits/55c1d5f688a8808e023f3e53d07cccf0fc70c0a6). What's our preference on keeping that `aria-disabled` attribute selector in the CSS? Or should I just revert[ that commit ](https://github.com/adobe/spectrum-css/pull/2886/commits/f023f57b643a1f2a50415c6fbc7b232684a8700f) and keep the CSS untouched?
```
&.is-disabled,
&[aria-disabled] // I haven't actually made this change yet
```

Reply:
I think adding a selector for aria-disabled might be a nice addition, instead of [disabled]. `aria-disabled` can equal false, so make sure that's `&[aria-disabled="true"]

Reply:
Lovely! here's the change: https://github.com/adobe/spectrum-css/pull/2886/commits/bf9b82eaa8f45364595657f0a4b8b9e4c93c4538